linkerd/admin/src/main/resources/io/buoyant/linkerd/admin/template/router_client.template

```diff
@@ -2,27 +2,58 @@
   {{client}}
 </div>
 
-<div class="client-metrics">
-  <dl>
-  {{#each metrics}}
+<div class="client-metrics col-md-4">
```

Comment:
hm, `col-md-*` classes really should be inside a `row`.
to avoid all the extra row/col classes, I'd make `.client-metrics` a row, then have 3 `.col-md-2` divs: one with requests/successes, one with connections/failures, and the last one with latencies